### ISSUE TITLE
Support right analog stick on Android

### ISSUE THREAD

**Issue by Andiweli** [EKA2L1/EKA2L1#532]
**Describe the bug**
Logitech G Cloud Handheld with built in controller and Android 11. Right analog stick can't be mapped. would be great for using strafe or looking in games.

- Android 11
- Snapdragon 720G
- Adreno 618

**Your emulator configuration**
N-Gage QD

**Comment by huchao2007** [comment]
Support right joystick mapping, many shooting games need it.

**Comment by mindverge** [comment]
Please add ability to map right analog stick, very needed for Ashen and Shadowkey

**Comment by Andiweli** [comment]
Huh is there an update on this? 😳 Project seems kinda dead.

**Comment by skilbirtpShrimpin** [comment]
I hope it's not dead, if I had a nickel for every time a android game emulation project had issues mapping the right joystick, I'd have two nickels, which isn't a lot but it's weird that it's happened twice

**Comment by Andiweli** [comment]
If somebody is interested I solved it by myself...

Everyone who want's to test the Dualstick EKA2L1 can try it https://github.com/Andiweli/EKA2L1-DualStick/releases/tag/v0.1.0

**Comment by mindverge** [comment]
Yes! thank you, finally I will replay gems like Ashen and Shadowkey with good controls. Setup is pita but it works in the end, great job!

**Comment by Andiweli** [comment]
> Yes! thank you, finally I will replay gems like Ashen and Shadowkey with good controls. Setup is pita but it works in the end, great job!

I've got another small update ready soon with improved analog stick recognition and other small fixes... eta today I think.
